terraform/global-resources/guardduty.tf

```diff
@@ -0,0 +1,448 @@
+#provider "aws" {
+#  region  = "${var.aws_region}"
+#  profile = "${var.aws_master_profile}"
+#}
```

Comment:
No need for this to be here (even commented out), should be dictated via `main.tf`